libp2p/Session.cpp

```diff
@@ -213,7 +199,6 @@ bool Session::checkPacket(bytesConstRef _msg)
 void Session::send(bytes&& _msg)
 {
     bytesConstRef msg(&_msg);
-    clog(VerbosityTrace, "net") << "<- " << RLP(msg.cropped(1));
```

Comment:
I think after removing this we don't log anything for outgoing packets of capabilities. Maybe leave it just saying that there's some outgoing packet? Probably you could also look at the first byte to get the packet type

Reply:
Good idea - we appear to log messages in other parts of the code when we perform actions which result in capability packets being sent (e.g. EthereumPeer::requestBlockHeaders) but it could also be nice to be able to track exactly when the capability packets are being sent and exactly which packets are being sent.